### ISSUE TITLE
ClipboardCopy icon missing in v2

### ISSUE THREAD

**Issue by wellbrained** [tailwindlabs/heroicons#751]
The new version has new clipboard icons but none of those really indicate the copy-function as it did in v1.

**Comment by noxify** [comment]
icons have been renamed.
Please check https://github.com/tailwindlabs/heroicons/issues/750#issuecomment-1225331419 to find the new name

**Comment by wellbrained** [comment]
Thanks for the info. I saw that table but as stated, the new icon doesn't fit or indicate the "copy" function as it did before.

**Comment by noxify** [comment]
maybe `arrow-left-on-rectangle` ?

**Comment by sschoger** [comment]
Hey! 

I decided to replace the arrow-on-clipboard with a more traditional copy to clipboard icon that has a document overlapping:
<img width="638" alt="CleanShot 2022-08-30 at 16 59 36@2x" src="https://user-images.githubusercontent.com/2652515/187542075-ab519433-3c77-48ef-999e-3b43069ba8b7.png">

When I was researching this icon I found this metaphor to be much more conventional.

Another more modern alternative that I've seen around the web is the stacked squares:
<img width="567" alt="CleanShot 2022-08-30 at 17 02 42@2x" src="https://user-images.githubusercontent.com/2652515/187542647-d1f82c60-8978-466d-9502-6092c1dff92a.png">

If you're looking for something thats closer to the original v1 icon I recommend using the `arrow-left-on-rectangle`:
<img width="656" alt="CleanShot 2022-08-30 at 17 04 13@2x" src="https://user-images.githubusercontent.com/2652515/187542985-fb4a5dfc-dd34-43e6-a61a-e37fd18b9af3.png">

Hope this helps clarify things!
